docs/preview/features/hosting/formatting.md

```diff
@@ -17,23 +17,39 @@ PM > Install-Package Arcus.WebApi.Hosting
 ## Restricting JSON format
 We have provided an extension that will allow you to restrict your input and output formatting to only JSON formatting (Only the `SystemTextJsonInputFormatter` will remain). This means that all other incoming content will result in `UnsupportedMediaType` failures and outgoing content will fail to serialize back to the sender. With this functionality, you'll be sure that you only have to deal with JSON.
 
-Following example shows you where you can configure this:
+Following example shows you where you can configure this in your `Program.cs`:
 
 ```csharp
 using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Mvc;
 
 WebApplicationBuilder builder = WebApplication.CreateBuilder();
 
-builder.Services.AddControllers(mvcOptions => 
+builder.Services.AddControllers(mvcOptions =>
 {
     mvcOptions.OnlyAllowJsonFormatting();
 });
 ```
 
+Example for .NET Core 5 and earlier
```

Comment:
Wasn't `AddControllers` not already introduced somewhere with .NET Core 3 ?
In other words, doesn't the first code sample really not work with .NET Core 3.x / .NET 5 ?

Reply:
My remark refers to the removal of Startup.cs
Not specifically to the existence of the AddControllers method.
.NET 6: https://docs.microsoft.com/en-us/aspnet/core/fundamentals/startup?view=aspnetcore-6.0
.NET 5: https://docs.microsoft.com/en-us/aspnet/core/fundamentals/startup?view=aspnetcore-5.0

Relating to AddControllers.
Yes this has been available from .NET Core 3.x
https://docs.microsoft.com/en-us/dotnet/api/microsoft.extensions.dependencyinjection.mvcservicecollectionextensions.addcontrollers?view=aspnetcore-3.0

Ben then I guess we could also argue that it could work in .NET Core 1.x by referring to the .NET 6 package?
https://docs.microsoft.com/en-us/dotnet/api/microsoft.extensions.dependencyinjection.mvcservicecollectionextensions.addcontrollers?view=aspnetcore-6.0&viewFallbackFrom=aspnetcore-1.0

Reply:
I guess the code sample would work with 3.x, and we only need to mention that it should be put in a different place?
And remove the old example.